osbuild/api.py

```diff
@@ -78,6 +84,9 @@ def __exit__(self, *args):
 
 class API(BaseAPI):
     """The main OSBuild API"""
+
+    endpoint = "osbuild"
```

Comment:
This works? You can override an ABC `@property` with an actual member field? Impressive.